WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -95,4 +128,35 @@
   });
 
   $('#display-<%= @display %>').button('toggle');
+
+  // From http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+  function getParameterByName(name, url) {
```

Comment:
Thanks for including a link to where you got this from. Can we move this into a `wca.getParameterByName` method in application.js so other part of our codebase could potentially use it?